### PR TITLE
Delete account-roles - delete both types for the default flow

### PR DESCRIPTION
**Pull request by oriAdler**
1. For `--hosted-cp` delete only hypershift account roles.
2. For `--classic` delete only classic account roles.

Related: [SDA-8666](https://issues.redhat.com/browse/SDA-8666)

Default flow deletes all account roles:
image

If role type is specified, delete only the adequate set:
image

### CODE REVIEW

File: cmd/dlt/accountroles/cmd.go [hunk 1]
```diff
@@ -66,6 +67,14 @@ func init() {
 	)
 	flags.MarkHidden("hosted-cp")
 
+	flags.BoolVar(
+		&args.hostedCP,
```
Comment (gdbranco) [inline cmd/dlt/accountroles/cmd.go:71]: classic

File: cmd/dlt/accountroles/cmd.go [hunk 2]
```diff
@@ -137,11 +146,24 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
-	// TODO: delete both classic and hosted CP roles by default, once managed policies are in place
-	err = deleteAccountRoles(r, env, prefix, clusters, mode, args.hostedCP)
-	if err != nil {
-		r.Reporter.Errorf("%s", err)
-		os.Exit(1)
+	// Default flow deletes both classic and hosted-CP roles
+	if args.classic == false && args.hostedCP == false {
```
Comment (gdbranco) [inline cmd/dlt/accountroles/cmd.go:150]: if !args.classic && !args.hostedCP

File: cmd/dlt/accountroles/cmd.go (hunk 1 above)
Comment (oriAdler) [reply cmd/dlt/accountroles/cmd.go]: Good catch!
Thanks

File: cmd/dlt/accountroles/cmd.go (hunk 2 above)
Comment (oriAdler) [reply cmd/dlt/accountroles/cmd.go]: Changed

**Pull request merged**